apps/protailwind/src/templates/workshop-template.tsx

```diff
@@ -89,45 +90,52 @@ const WorkshopTemplate: React.FC<{
           )}
         </div>
 
-        {hasPurchased ? (
+        {!commersePropsLoading && (
```

Comment:
@DrShpongle would it be possible to avoid layout shift while still showing the loading skeleton? I think it's better to indicate that something is loading within the sidebar area instead of leaving it empty.